### PR TITLE
[SUCE-228]FE - SUTTA VIEW tools side by side text view option

### PR DESCRIPTION
**Pull request by TrebuhD**

### CODE REVIEW

File: client/elements/sc-donate-now-page.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (gradam) [inline client/elements/sc-donate-now-page.html]: Do we really need this line?

File: client/elements/text/sc-segmented-text.html [hunk 2]
```diff
@@ -259,66 +272,99 @@
...
+                  case 'sidebyside':
+                      textContent.classList.add('side-by-side');
+                      this._addSecondaryText();
+                      break;
+                  case 'linebyline':
+                      textContent.classList.add('line-by-line');
+                      this._addSecondaryText();
+                      break;
+                  case 'popup':
+                      Array.from(textContent.querySelectorAll('.translated-text')).forEach(item =>
```
Comment (gradam) [inline client/elements/text/sc-segmented-text.html]: It is a better practice to have up to 2 lines in switch/case (extract to function)

File: client/elements/text/sc-segmented-text.html [hunk 3]
```diff
@@ -259,66 +272,99 @@
...
+          // reset all classes and remove tooltips.
+          _resetViewOptions() {
+              this.$.segmented_text_content.classList.remove('side-by-side', 'line-by-line', 'show-pali');
+              Array.from(this.shadowRoot.querySelectorAll('paper-tooltip'))
+                  .forEach(item => item.parentNode.removeChild(item));
+          }
+
+          _addPrimaryText(textStrings) {
+              try {
+                  const textContainer = this.shadowRoot.querySelector('#segmented_text_content');
```
Comment (gradam) [inline client/elements/text/sc-segmented-text.html]: maybe you can extract something from this try/catch?

File: client/elements/text/sc-segmented-text.html [hunk 4]
```diff
@@ -407,37 +413,55 @@
...
                       .remove('latin-script', 'sinhala-script', 'devanagari-script', 'thai-script', 'myanmar-script');
                   Array.from(segments).forEach(item => item.innerHTML = this.suttaOriginalText[item.id] + ' ');
                   Array.from(tooltips).forEach(item => item.innerHTML = this.suttaOriginalText[item.id]);
                   switch (paliScript) {
                       case 0:
-                          this.$.html_text_content.classList.add('latin-script');
+                          this.$.segmented_text_content.classList.add('latin-script');
                           break;
                       case 1:
-                          this.$.html_text_content.classList.add('sinhala-script');
```
Comment (gradam) [inline client/elements/text/sc-segmented-text.html]: those 4 lines in cases[1-4] can be extracted to separate function

File: server/server/common/queries.py [hunk 5]
```diff
@@ -255,16 +242,38 @@
...
+        FILTER object.uid == @uid AND object.lang == root_text.root_lang
+        LIMIT 1 
+        RETURN object
+)[0]
+
+LET translated_po_obj = (
+    FOR object IN po_strings 
+        FILTER object.uid == @uid AND object.lang == @language AND LOWER(object.author) == @author
+        LIMIT 1 
+        RETURN object
```
Comment (gradam) [inline server/server/common/queries.py]: it returns unnecessary information

File: server/server/common/queries.py [hunk 6]
```diff
@@ -255,16 +242,38 @@
...
+        FILTER markup.uid == @uid
+        LIMIT 1
+        RETURN markup.markup
+)[0]
+
+LET root_po_obj = (
+    FOR object IN po_strings
+        FILTER object.uid == @uid AND object.lang == root_text.root_lang
+        LIMIT 1 
+        RETURN object
```
Comment (gradam) [inline server/server/common/queries.py]: it returns unnecessary information

File: client/elements/sc-donate-now-page.html (hunk 1 above)
Comment (TrebuhD) [reply client/elements/sc-donate-now-page.html]: Yes. I'm really attached to it.

Comment (gradam) [reply client/elements/sc-donate-now-page.html]: in that case. let it be :)

**Pull request merged**